src/views/List.jsx

```diff
@@ -57,10 +73,17 @@ export function List({ data, userId }) {
 			</form>
 
 			<ul>
-				{filteredList &&
-					filteredList.map((item) => {
-						return <ListItem key={item.id} item={item} />;
-					})}
+				{filteredObject &&
+					Object.entries(filteredObject).map(([timeBucket, list]) => (
```

Comment:
I like the implementation of using Object.entries here. I definitely learned a bit about data type conversion by looking at this.